.github/workflows/docker.yml

```diff
@@ -94,7 +96,7 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       -
         name: Build and push
-        if: ${{ (github.event_name == 'workflow_dispatch' && contains(github.event.inputs.message, '[main]')) }}
+        if: ${{ env.SHOULD_RUN === 'true' | (github.event_name == 'workflow_dispatch' && contains(github.event.inputs.message, '[main]')) }}
```

Comment:
Also, why is this required? There's already a GH Action id called `should_run` - won't that confuse future maintainers?

Reply:
Rather than use it like an env var, you can use it like https://github.com/ferdium/ferdium-app/blob/develop/.github/workflows/builds.yml#L108

Reply:
> I think it should be 2 vertical pipe symbols? `||`?

Yeah, I think so

Reply:
> Also, why is this required? There's already a GH Action id called `should_run` - won't that confuse future maintainers?

This is required because the output of that actions should trigger the build and publish (it there are changes, which didn't happen on the latest commit change.